src/app/heroes/heroes.component.html

```diff
@@ -0,0 +1,19 @@
+<div>
+  <label>Hero name:
+    <input #heroName />
+  </label>
```

Comment:
The input tag should have `name` and `type` as well as `id` attributes in general. Also, Label tags should have `for` attribute that looking for the input related.